processing/src/test/java/io/druid/segment/incremental/IncrementalIndexTest.java

```diff
@@ -88,6 +90,49 @@ public IncrementalIndex createIndex()
                     );
                   }
                 }
+            },
+            {
+                new IndexCreator()
+                {
+                  @Override
+                  public IncrementalIndex createIndex()
+                  {
+                    return new OnheapIncrementalIndex(
+                        0,
+                        QueryGranularity.MINUTE,
+                        new AggregatorFactory[]{
+                            new FilteredAggregatorFactory(
+                                new CountAggregatorFactory("cnt"),
```

Comment:
can u add FilteredAggregator to existing set of indexes instead of adding a new one just for this test?